client/java/kserve-api/src/main/proto/grpc_predict_v2.proto

```diff
@@ -0,0 +1,328 @@
+// Copyright 2020 kubeflow.org.
```

Comment:
2022?

Reply:
copied from https://github.com/openvinotoolkit/model_server/blob/main/src/kfserving_api/grpc_predict_v2.proto

Reply:
Please update.

Reply:
AFAIK we update dates in copyright headers.
Please consult it with SDL owner: Jarosław Szczepański

Reply:
to be replaced with acquisition thru wget